rotkehlchen/data_import/importers/bitmex.py

```diff
@@ -0,0 +1,146 @@
+import csv
+import logging
+from enum import Enum
+from pathlib import Path
+from typing import Any
+
+from rotkehlchen.accounting.structures.balance import AssetBalance, Balance
+from rotkehlchen.constants import ZERO
+from rotkehlchen.constants.assets import A_BTC, A_USD
+from rotkehlchen.data_import.utils import BaseExchangeImporter, UnsupportedCSVEntry
+from rotkehlchen.db.drivers.gevent import DBCursor
+from rotkehlchen.errors.misc import InputError
+from rotkehlchen.errors.serialization import DeserializationError
+from rotkehlchen.exchanges.data_structures import AssetMovement, MarginPosition
+from rotkehlchen.exchanges.utils import deserialize_asset_movement_address, get_key_if_has_val
+from rotkehlchen.history.price import PriceHistorian
+from rotkehlchen.logging import RotkehlchenLogsAdapter
+from rotkehlchen.serialization.deserialize import (
+    deserialize_asset_amount,
+    deserialize_asset_amount_force_positive,
+    deserialize_fee,
+    deserialize_timestamp_from_date,
+)
+from rotkehlchen.types import AssetAmount, AssetMovementCategory, Fee, Location
+from rotkehlchen.utils.misc import satoshis_to_btc
+
+logger = logging.getLogger(__name__)
+log = RotkehlchenLogsAdapter(logger)
+
+
+class BitMEXWalletHistoryColnames(Enum):
```

Comment:
Is this Enum used anywhere?

Reply:
I added it after typing out all the column names when accessing `csv_row` but I never ended up using it. Do you think it's better to use something like this or just access columns of `csv_row` by typing them out directly?

Reply:
> I added it after typing out all the column names when accessing `csv_row` but I never ended up using it. Do you think it's better to use something like this or just access columns of `csv_row` by typing them out directly?

Hmmm if it's used in multiple places maybe this is not bad, but using them directly if they are just used in the parser is fine. No need to create in memory structures like this enum which will persists for the entire lifetime of the app.

Reply:
ok, thanks. I wasn't sure so good to know. I'll remove it, squash my commits and push again